pkg/config/config.go

```diff
@@ -14,9 +14,26 @@
 package config
 
 import (
+	"os"
+
 	"github.com/kelseyhightower/envconfig"
+	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+var log = ctrl.Log.WithName("setup")
+
+//ControllerCfg is a global variable to keep the configuration for Chaos Controller
+var ControllerCfg *ChaosControllerConfig
+
+func init() {
+	conf, err := EnvironChaosController()
+	if err != nil {
+		log.Error(err, "Chaos Controller: invalid environment configuration")
+		os.Exit(1)
+	}
+	ControllerCfg = &conf
+}
```

Comment:
I don't think this is correct. If you want to use `init()` here, please separate the config package for chaos-dashboard and chaos-controller. Otherwise in the chaos-dashboard component it will execute this function, too

Reply:
Thank you. I have fixed it. PTAL

Reply:
The `init` function is package level. So even you separate the go file for dashboard and controller, the init function will still be executed. https://www.digitalocean.com/community/tutorials/understanding-init-in-go

I recommend we don't use init function in the config package. What about using a global var to keep `ControllerConfig` in `controllers/common/types.go`?

Reply:
@yeya24 
My mistake, I made a mistake in my test environment, I used an earlier version of Dashboard.
I have fixed it. PTAL. Thank you.